k2/csrc/algorithms.h

```diff
@@ -134,7 +137,8 @@ class Renumbering {
   Array1<char> keep_;  // array of elements to keep; dimension is the
                        // `num_old_elems` provided in the constructor but it
                        // was allocated with one extra element.
-  Array1<int32_t> old2new_;  // note: actual dimension is num-old-elems + 1.
+  Array1<int32_t> old2new_;  // note: dimension is num-old-elems + 1.
+  Array1<int32_t> old2new_part_;  // range of old2new_, of dim num-old-elems.
```

Comment:
It is never used.

I guess you were planning to use it in `Old2New(bool extra_element = false)`